benchexec/tools/dartagnan.py

```diff
@@ -0,0 +1,47 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import xml.etree.ElementTree as ET
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for Dartagnan (http://https://github.com/hernanponcedeleon/Dat3M).
+    """
+
+    def executable(self):
+        return util.find_executable("java")
+
+    def name(self):
+        return "Dartagnan"
+
+    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
+        return [executable, "-jar", "svcomp/target/svcomp-2.0.4-jar-with-dependencies.jar"] + options + tasks
```

Comment:
This line hard-codes a version number, which means that this tool-info wouldn't work with any different version of your tool. Could you make this more flexible such that this works with all tool versions?

Also, this works only if the user executes `benchexec` exactly in the correct directory (the parent directory of the `svcomp` directory). Is the directory structure `svcomp/target/` part of your tool distribution, e.g., your ZIP archive for SV-COMP? And is the parent directory the most obvious directory for the user to be in? Otherwise the tool-info module could for example look for the JAR file in the current directory and subdirectories.